Shaun Develop Breaks on Browser Load

Potentially, upgrading to MongoDB 4 has broken the GitPod Development.

I am looking into it, but here are the errors:

Shortened Error (me trying to restart Mongodb, after an initial crash):

```console
gitpod /workspace/freeCodeCamp $ mongod
{"t":{"$date":"2021-03-25T20:17:06.840+00:00"},"s":"I",  "c":"CONTROL",  "id":23285,   "ctx":"main","msg":"Automatically disabling TLS 1.0, to force-enable TLS 1.0 specify --sslDisabledProtocols 'none'"}
{"t":{"$date":"2021-03-25T20:17:06.842+00:00"},"s":"W",  "c":"ASIO",     "id":22601,   "ctx":"main","msg":"No TransportLayer configured during NetworkInterface startup"}
{"t":{"$date":"2021-03-25T20:17:06.842+00:00"},"s":"I",  "c":"NETWORK",  "id":4648601, "ctx":"main","msg":"Implicit TCP FastOpen unavailable. If TCP FastOpen is required, set tcpFastOpenServer, tcpFastOpenClient, and tcpFastOpenQueueSize."}
{"t":{"$date":"2021-03-25T20:17:06.842+00:00"},"s":"I",  "c":"STORAGE",  "id":4615611, "ctx":"initandlisten","msg":"MongoDB starting","attr":{"pid":5489,"port":27017,"dbPath":"/data/db","architecture":"64-bit","host":"ws-09fd1a75-ab9b-4403-9cbc-022f6cd3bff1"}}
{"t":{"$date":"2021-03-25T20:17:06.842+00:00"},"s":"W",  "c":"CONTROL",  "id":20720,   "ctx":"initandlisten","msg":"Available memory is less than system memory","attr":{"availableMemSizeMB":12288,"systemMemSizeMB":64322}}
```

<details>

  <summary>mongod --version</summary>

```console
gitpod /workspace/freeCodeCamp $ mongod --version
db version v4.4.2
Build Info: {
    "version": "4.4.2",
    "gitVersion": "15e73dc5738d2278b688f8929aee605fe4279b0e",
    "openSSLVersion": "OpenSSL 1.1.1f  31 Mar 2020",
    "modules": [],
    "allocator": "tcmalloc",
    "environment": {
        "distmod": "ubuntu2004",
        "distarch": "x86_64",
        "target_arch": "x86_64"
    }
}
```
</details>

<details>
 
  <summary>Full Error</summary>

```console
{"t":{"$date":"2021-03-25T20:13:18.057+00:00"},"s":"I",  "c":"NETWORK",  "id":22943,   "ctx":"listener","msg":"Connection accepted","attr":{"remote":"127.0.0.1:33936","connectionId":1,"connectionCount":1}}
{"t":{"$date":"2021-03-25T20:13:18.060+00:00"},"s":"I",  "c":"NETWORK",  "id":51800,   "ctx":"conn1","msg":"client metadata","attr":{"remote":"127.0.0.1:33936","client":"conn1","doc":{"driver":{"name":"nodejs","version":"3.5.5"},"os":{"type":"Linux","name":"linux","architecture":"x64","version":"5.4.0-1024-gcp"},"platform":"'Node.js v14.16.0, LE (legacy)"}}}
{"t":{"$date":"2021-03-25T20:13:18.071+00:00"},"s":"I",  "c":"STORAGE",  "id":20320,   "ctx":"conn1","msg":"createCollection","attr":{"namespace":"freecodecamp.user","uuidDisposition":"generated","uuid":{"uuid":{"$uuid":"fbec89a6-62e6-4a86-a1e4-c4c3ec7b6417"}},"options":{}}}
{"t":{"$date":"2021-03-25T20:13:18.072+00:00"},"s":"I",  "c":"NETWORK",  "id":22943,   "ctx":"listener","msg":"Connection accepted","attr":{"remote":"127.0.0.1:33942","connectionId":2,"connectionCount":2}}
{"t":{"$date":"2021-03-25T20:13:18.072+00:00"},"s":"I",  "c":"NETWORK",  "id":51800,   "ctx":"conn2","msg":"client metadata","attr":{"remote":"127.0.0.1:33942","client":"conn2","doc":{"driver":{"name":"nodejs","version":"3.5.5"},"os":{"type":"Linux","name":"linux","architecture":"x64","version":"5.4.0-1024-gcp"},"platform":"'Node.js v14.16.0, LE (legacy)"}}}
{"t":{"$date":"2021-03-25T20:13:18.073+00:00"},"s":"I",  "c":"STORAGE",  "id":20320,   "ctx":"conn2","msg":"createCollection","attr":{"namespace":"freecodecamp.user","uuidDisposition":"generated","uuid":{"uuid":{"$uuid":"196acb64-73d1-4a46-9244-a49a9437ead6"}},"options":{}}}
{"t":{"$date":"2021-03-25T20:13:18.073+00:00"},"s":"I",  "c":"NETWORK",  "id":22943,   "ctx":"listener","msg":"Connection accepted","attr":{"remote":"127.0.0.1:33944","connectionId":3,"connectionCount":3}}
{"t":{"$date":"2021-03-25T20:13:18.074+00:00"},"s":"I",  "c":"NETWORK",  "id":51800,   "ctx":"conn3","msg":"client metadata","attr":{"remote":"127.0.0.1:33944","client":"conn3","doc":{"driver":{"name":"nodejs","version":"3.5.5"},"os":{"type":"Linux","name":"linux","architecture":"x64","version":"5.4.0-1024-gcp"},"platform":"'Node.js v14.16.0, LE (legacy)"}}}
{"t":{"$date":"2021-03-25T20:13:18.182+00:00"},"s":"I",  "c":"INDEX",    "id":20345,   "ctx":"conn1","msg":"Index build: done building","attr":{"buildUUID":null,"namespace":"freecodecamp.user","index":"_id_","commitTimestamp":{"$timestamp":{"t":0,"i":0}}}}
{"t":{"$date":"2021-03-25T20:13:18.182+00:00"},"s":"I",  "c":"COMMAND",  "id":51803,   "ctx":"conn1","msg":"Slow query","attr":{"type":"command","ns":"freecodecamp.user","command":{"insert":"user","documents":[{"_id":{"$oid":"5bd30e0f1caf6ac3ddddddb5"},"email":"foo@bar.com","emailVerified":true,"progressTimestamps":[],"isBanned":false,"isCheater":false,"username":"developmentuser","about":"","name":"Development User","location":"","picture":"https://freecodecamp.com/sample-image.png","acceptedPrivacyTerms":true,"sendQuincyEmail":false,"currentChallengeId":"","isHonest":false,"isFrontEndCert":false,"isDataVisCert":false,"isBackEndCert":false,"isFullStackCert":false,"isRespWebDesignCert":false,"is2018DataVisCert":false,"isFrontEndLibsCert":false,"isJsAlgoDataStructCert":false,"isApisMicroservicesCert":false,"isInfosecQaCert":false,"isQaCertV7":false,"isInfosecCertV7":false,"is2018FullStackCert":false,"isSciCompPyCertV7":false,"isDataAnalysisPyCertV7":false,"isMachineLearningPyCertV7":false,"completedChallenges":[],"portfolio":[],"yearsTopContributor":[],"rand":0.6126749173148205,"theme":"default","profileUI":{"isLocked":true,"showAbout":false,"showCerts":false,"showDonation":false,"showHeatMap":false,"showLocation":false,"showName":false,"showPoints":false,"showPortfolio":false,"showTimeLine":false},"badges":{"coreTeam":[]},"isDonating":false,"emailAuthLinkTTL":null,"emailVerifyTTL":null}],"ordered":true,"lsid":{"id":{"$uuid":"1c249d4c-c6b7-439b-a4f9-037d9b05e0da"}},"$db":"freecodecamp"},"ninserted":1,"keysInserted":1,"numYields":0,"reslen":45,"locks":{"ParallelBatchWriterMode":{"acquireCount":{"r":5}},"ReplicationStateTransition":{"acquireCount":{"w":5}},"Global":{"acquireCount":{"r":2,"w":3}},"Database":{"acquireCount":{"r":2,"w":3}},"Collection":{"acquireCount":{"r":1,"w":3}},"Mutex":{"acquireCount":{"r":6}}},"flowControl":{"acquireCount":3,"timeAcquiringMicros":2},"storage":{},"protocol":"op_msg","durationMillis":110}}
{"t":{"$date":"2021-03-25T20:13:18.217+00:00"},"s":"I",  "c":"STORAGE",  "id":20279,   "ctx":"conn2","msg":"Conflicted creating a collection","attr":{"namespace":"freecodecamp.user","uuid":{"uuid":{"$uuid":"196acb64-73d1-4a46-9244-a49a9437ead6"}}}}
{"t":{"$date":"2021-03-25T20:13:18.232+00:00"},"s":"I",  "c":"COMMAND",  "id":51803,   "ctx":"conn2","msg":"Slow query","attr":{"type":"command","ns":"freecodecamp.user","command":{"insert":"user","documents":[{"_id":{"$oid":"5bd30e0f1caf6ac3ddddddb9"},"email":"bar@bar.com","emailVerified":true,"progressTimestamps":[],"isBanned":false,"isCheater":false,"username":"twaha","about":"","name":"Development User","location":"","picture":"https://freecodecamp.com/sample-image.png","acceptedPrivacyTerms":true,"sendQuincyEmail":false,"currentChallengeId":"","isHonest":false,"isFrontEndCert":false,"isDataVisCert":false,"isBackEndCert":false,"isFullStackCert":false,"isRespWebDesignCert":false,"is2018DataVisCert":false,"isFrontEndLibsCert":false,"isJsAlgoDataStructCert":false,"isApisMicroservicesCert":false,"isInfosecQaCert":false,"isQaCertV7":false,"isInfosecCertV7":false,"is2018FullStackCert":false,"isSciCompPyCertV7":false,"isDataAnalysisPyCertV7":false,"isMachineLearningPyCertV7":false,"completedChallenges":[],"portfolio":[],"yearsTopContributor":[],"rand":0.6126749173148205,"theme":"default","profileUI":{"isLocked":true,"showAbout":false,"showCerts":false,"showDonation":false,"showHeatMap":false,"showLocation":false,"showName":false,"showPoints":false,"showPortfolio":false,"showTimeLine":false},"badges":{"coreTeam":[]},"isDonating":false,"emailAuthLinkTTL":null,"emailVerifyTTL":null}],"ordered":true,"lsid":{"id":{"$uuid":"5ca869ad-cd40-4a77-9cd4-2ecb8ea5635f"}},"$db":"freecodecamp"},"ninserted":1,"keysInserted":1,"writeConflicts":1,"numYields":0,"reslen":45,"locks":{"ParallelBatchWriterMode":{"acquireCount":{"r":6}},"ReplicationStateTransition":{"acquireCount":{"w":6}},"Global":{"acquireCount":{"r":2,"w":4}},"Database":{"acquireCount":{"r":2,"w":4}},"Collection":{"acquireCount":{"r":1,"w":4}},"Mutex":{"acquireCount":{"r":6}}},"flowControl":{"acquireCount":4,"timeAcquiringMicros":4},"storage":{},"protocol":"op_msg","durationMillis":159}}
{"t":{"$date":"2021-03-25T20:13:18.234+00:00"},"s":"I",  "c":"NETWORK",  "id":22944,   "ctx":"conn2","msg":"Connection ended","attr":{"remote":"127.0.0.1:33942","connectionId":2,"connectionCount":2}}
{"t":{"$date":"2021-03-25T20:13:18.234+00:00"},"s":"I",  "c":"NETWORK",  "id":22944,   "ctx":"conn1","msg":"Connection ended","attr":{"remote":"127.0.0.1:33936","connectionId":1,"connectionCount":1}}
{"t":{"$date":"2021-03-25T20:13:18.234+00:00"},"s":"I",  "c":"NETWORK",  "id":22944,   "ctx":"conn3","msg":"Connection ended","attr":{"remote":"127.0.0.1:33944","connectionId":3,"connectionCount":0}}
{"t":{"$date":"2021-03-25T20:13:18.480+00:00"},"s":"I",  "c":"NETWORK",  "id":22943,   "ctx":"listener","msg":"Connection accepted","attr":{"remote":"127.0.0.1:33972","connectionId":4,"connectionCount":1}}
{"t":{"$date":"2021-03-25T20:13:18.480+00:00"},"s":"I",  "c":"NETWORK",  "id":51800,   "ctx":"conn4","msg":"client metadata","attr":{"remote":"127.0.0.1:33972","client":"conn4","doc":{"application":{"name":"MongoDB Shell"},"driver":{"name":"MongoDB Internal Client","version":"4.4.2"},"os":{"type":"Linux","name":"Ubuntu","architecture":"x86_64","version":"20.04"}}}}
{"t":{"$date":"2021-03-25T20:13:18.483+00:00"},"s":"I",  "c":"COMMAND",  "id":20461,   "ctx":"conn4","msg":"CMD fsync","attr":{"lock":true}}
{"t":{"$date":"2021-03-25T20:13:18.645+00:00"},"s":"W",  "c":"COMMAND",  "id":20471,   "ctx":"fsyncLockWorker","msg":"WARNING: instance is locked, blocking all writes. The fsync command has finished execution, remember to unlock the instance using fsyncUnlock()."}
{"t":{"$date":"2021-03-25T20:13:18.645+00:00"},"s":"I",  "c":"COMMAND",  "id":20462,   "ctx":"conn4","msg":"mongod is locked and no writes are allowed","attr":{"lockCount":1,"seeAlso":"http://dochub.mongodb.org/core/fsynccommand"}}
{"t":{"$date":"2021-03-25T20:13:18.646+00:00"},"s":"I",  "c":"COMMAND",  "id":51803,   "ctx":"conn4","msg":"Slow query","attr":{"type":"command","ns":"admin.$cmd","appName":"MongoDB Shell","command":{"fsync":1.0,"lock":true,"lsid":{"id":{"$uuid":"9ec4ae4b-ed3d-4d8d-898f-fa0d9f99fba0"}},"$db":"admin"},"numYields":0,"reslen":182,"locks":{"Mutex":{"acquireCount":{"W":1}}},"protocol":"op_msg","durationMillis":162}}
{"t":{"$date":"2021-03-25T20:13:18.646+00:00"},"s":"I",  "c":"COMMAND",  "id":20465,   "ctx":"conn4","msg":"command: unlock requested"}
{"t":{"$date":"2021-03-25T20:13:18.658+00:00"},"s":"I",  "c":"COMMAND",  "id":20466,   "ctx":"conn4","msg":"fsyncUnlock complete","attr":{"lockCount":0}}
{"t":{"$date":"2021-03-25T20:13:18.661+00:00"},"s":"I",  "c":"NETWORK",  "id":22944,   "ctx":"conn4","msg":"Connection ended","attr":{"remote":"127.0.0.1:33972","connectionId":4,"connectionCount":0}}
{"t":{"$date":"2021-03-25T20:13:36.765+00:00"},"s":"I",  "c":"NETWORK",  "id":22943,   "ctx":"listener","msg":"Connection accepted","attr":{"remote":"127.0.0.1:35136","connectionId":5,"connectionCount":1}}
{"t":{"$date":"2021-03-25T20:13:36.765+00:00"},"s":"I",  "c":"NETWORK",  "id":22943,   "ctx":"listener","msg":"Connection accepted","attr":{"remote":"127.0.0.1:35138","connectionId":6,"connectionCount":2}}
{"t":{"$date":"2021-03-25T20:13:36.765+00:00"},"s":"I",  "c":"NETWORK",  "id":22943,   "ctx":"listener","msg":"Connection accepted","attr":{"remote":"127.0.0.1:35140","connectionId":7,"connectionCount":3}}
{"t":{"$date":"2021-03-25T20:13:36.772+00:00"},"s":"I",  "c":"NETWORK",  "id":51800,   "ctx":"conn5","msg":"client metadata","attr":{"remote":"127.0.0.1:35136","client":"conn5","doc":{"driver":{"name":"nodejs","version":"3.6.3"},"os":{"type":"Linux","name":"linux","architecture":"x64","version":"5.4.0-1024-gcp"},"platform":"'Node.js v14.16.0, LE (legacy)"}}}
{"t":{"$date":"2021-03-25T20:13:36.772+00:00"},"s":"I",  "c":"NETWORK",  "id":51800,   "ctx":"conn6","msg":"client metadata","attr":{"remote":"127.0.0.1:35138","client":"conn6","doc":{"driver":{"name":"nodejs","version":"3.6.3"},"os":{"type":"Linux","name":"linux","architecture":"x64","version":"5.4.0-1024-gcp"},"platform":"'Node.js v14.16.0, LE (legacy)"}}}
{"t":{"$date":"2021-03-25T20:13:36.775+00:00"},"s":"I",  "c":"NETWORK",  "id":51800,   "ctx":"conn7","msg":"client metadata","attr":{"remote":"127.0.0.1:35140","client":"conn7","doc":{"driver":{"name":"nodejs","version":"3.6.3"},"os":{"type":"Linux","name":"linux","architecture":"x64","version":"5.4.0-1024-gcp"},"platform":"'Node.js v14.16.0, LE (unified)"}}}
{"t":{"$date":"2021-03-25T20:13:36.796+00:00"},"s":"I",  "c":"NETWORK",  "id":22943,   "ctx":"listener","msg":"Connection accepted","attr":{"remote":"127.0.0.1:35142","connectionId":8,"connectionCount":4}}
{"t":{"$date":"2021-03-25T20:13:36.797+00:00"},"s":"I",  "c":"NETWORK",  "id":51800,   "ctx":"conn8","msg":"client metadata","attr":{"remote":"127.0.0.1:35142","client":"conn8","doc":{"driver":{"name":"nodejs","version":"3.6.3"},"os":{"type":"Linux","name":"linux","architecture":"x64","version":"5.4.0-1024-gcp"},"platform":"'Node.js v14.16.0, LE (unified)"}}}
{"t":{"$date":"2021-03-25T20:13:36.798+00:00"},"s":"I",  "c":"STORAGE",  "id":20320,   "ctx":"conn8","msg":"createCollection","attr":{"namespace":"freecodecamp.sessions","uuidDisposition":"generated","uuid":{"uuid":{"$uuid":"85bb3494-7165-4e36-a25d-fdfa45918b0d"}},"options":{}}}
{"t":{"$date":"2021-03-25T20:13:36.900+00:00"},"s":"I",  "c":"INDEX",    "id":20345,   "ctx":"conn8","msg":"Index build: done building","attr":{"buildUUID":null,"namespace":"freecodecamp.sessions","index":"_id_","commitTimestamp":{"$timestamp":{"t":0,"i":0}}}}
{"t":{"$date":"2021-03-25T20:13:36.900+00:00"},"s":"I",  "c":"INDEX",    "id":20345,   "ctx":"conn8","msg":"Index build: done building","attr":{"buildUUID":null,"namespace":"freecodecamp.sessions","index":"expires_1","commitTimestamp":{"$timestamp":{"t":0,"i":0}}}}
{"t":{"$date":"2021-03-25T20:13:36.900+00:00"},"s":"I",  "c":"COMMAND",  "id":51803,   "ctx":"conn8","msg":"Slow query","attr":{"type":"command","ns":"freecodecamp.sessions","command":{"createIndexes":"sessions","indexes":[{"name":"expires_1","key":{"expires":1},"expireAfterSeconds":0}],"lsid":{"id":{"$uuid":"82885bda-9207-4194-bcc5-e2f50d3aa14a"}},"$db":"freecodecamp"},"numYields":0,"reslen":114,"locks":{"ParallelBatchWriterMode":{"acquireCount":{"r":5}},"ReplicationStateTransition":{"acquireCount":{"w":5}},"Global":{"acquireCount":{"r":2,"w":3}},"Database":{"acquireCount":{"r":2,"w":3}},"Collection":{"acquireCount":{"r":4,"w":2}},"Mutex":{"acquireCount":{"r":5}}},"flowControl":{"acquireCount":1,"timeAcquiringMicros":1},"storage":{},"protocol":"op_msg","durationMillis":101}}
{"t":{"$date":"2021-03-25T20:13:47.301+00:00"},"s":"I",  "c":"NETWORK",  "id":22943,   "ctx":"listener","msg":"Connection accepted","attr":{"remote":"127.0.0.1:35840","connectionId":9,"connectionCount":5}}
{"t":{"$date":"2021-03-25T20:13:47.301+00:00"},"s":"I",  "c":"NETWORK",  "id":51800,   "ctx":"conn9","msg":"client metadata","attr":{"remote":"127.0.0.1:35840","client":"conn9","doc":{"driver":{"name":"nodejs","version":"3.6.3"},"os":{"type":"Linux","name":"linux","architecture":"x64","version":"5.4.0-1024-gcp"},"platform":"'Node.js v14.16.0, LE (unified)"}}}
```
</details>


<details>

  <summary>Potentially relevant client error</summary>

```console
To create a production build, use gatsby build
⠀
warn Critical dependency: the request of a dependency is an
expression
warn Critical dependency: the request of a dependency is an
expression
warn chunk commons [mini-css-extract-plugin]
Conflicting order. Following module has been added:
 * css ./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[9].one
Of[1].use[1]!./node_modules/gatsby-plugin-postcss/node_modules/postcs
s-loader/dist/cjs.js??ruleSet[1].rules[9].oneOf[1].use[2]!./src/templ
ates/Challenges/components/completion-modal.css
despite it was not able to fulfill desired ordering with these
modules:
 * css ./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[9].one
Of[1].use[1]!./node_modules/gatsby-plugin-postcss/node_modules/postcs
s-loader/dist/cjs.js??ruleSet[1].rules[9].oneOf[1].use[2]!./src/templ
ates/Challenges/projects/tool-panel.css
   - couldn't fulfill desired order of chunk group(s)
component---src-templates-challenges-projects-frontend-show-js
   - while fulfilling desired order of chunk group(s)
component---src-templates-challenges-projects-backend-show-js
warn chunk commons [mini-css-extract-plugin]
Conflicting order. Following module has been added:
 * css ./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[9].one
Of[1].use[1]!./node_modules/gatsby-plugin-postcss/node_modules/postcs
s-loader/dist/cjs.js??ruleSet[1].rules[9].oneOf[1].use[2]!./src/templ
ates/Challenges/components/help-modal.css
despite it was not able to fulfill desired ordering with these
modules:
 * css ./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[9].one
Of[1].use[1]!./node_modules/gatsby-plugin-postcss/node_modules/postcs
s-loader/dist/cjs.js??ruleSet[1].rules[9].oneOf[1].use[2]!./src/templ
ates/Challenges/projects/tool-panel.css
   - couldn't fulfill desired order of chunk group(s)
component---src-templates-challenges-projects-frontend-show-js
   - while fulfilling desired order of chunk group(s)
component---src-templates-challenges-projects-backend-show-js
success Building development bundle - 119.147s

Couldn't read the file /workspace/freeCodeCamp/client, possibly due to source maps failing
original error Error: ENOENT: no such file or directory, open '//public/webpack.stats.json'
    at Object.openSync (fs.js:476:3)
    at Object.fs [as readFileSync] (fs.js:377:35)
    at /workspace/freeCodeCamp/client/public/webpack:/@freecodecamp/client/.cache/ssr-develop-static-entry.js:24:3
    at /workspace/freeCodeCamp/client/public/render-page.js:410298:3
    at Object.<anonymous> (/workspace/freeCodeCamp/client/public/render-page.js:410303:12)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: '//public/webpack.stats.json'
Failed to validate error [Error [ValidationError]: "filePath" is not
allowed to be empty] {
  _original: {
    context: { path: '/', filePath: '', line: undefined, column:
undefined },
    filePath: '',
    location: { start: [Object] },
    text: 'The path "/" errored during SSR.\n' +
      '\n' +
      '    Edit its component  to resolve the error.',
    level: 'WARNING',
    docsUrl: 'https://gatsby.dev/debug-html',
    stack: [],
    code: '11614'
  },
  details: [
    {
      message: '"filePath" is not allowed to be empty',
      path: [Array],
      type: 'string.empty',
      context: [Object]
    }
  ]
}

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @freecodecamp/client@1.0.0 develop: `gatsby develop --inspect=9230 "-H" "0.0.0.0"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @freecodecamp/client@1.0.0 develop script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/gitpod/.npm/_logs/2021-03-25T20_43_43_034Z-debug.log
```
</details>

- Disabled Feature Preview
- Rebased fork against upstream

<details>
  <summary>New Client Error (similar, but different)</summary>

```console
success Building development bundle - 99.732s

Couldn't read the file /workspace/freeCodeCamp/client, possibly due to source maps failing
original error Error: ENOENT: no such file or directory, open '//public/webpack.stats.json'
    at Object.openSync (fs.js:476:3)
    at Object.fs [as readFileSync] (fs.js:377:35)
    at /workspace/freeCodeCamp/client/public/webpack:/@freecodecamp/client/.cache/ssr-develop-static-entry.js:24:3
    at /workspace/freeCodeCamp/client/public/render-page.js:410321:3
    at Object.<anonymous> (/workspace/freeCodeCamp/client/public/render-page.js:410326:12)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: '//public/webpack.stats.json'
Failed to validate error [Error [ValidationError]: "type" must be one of
[GRAPHQL, CONFIG, WEBPACK, PLUGIN]] {
  _original: {
    '0': 'E',
    '1': 'N',
    '2': 'O',
    '3': 'E',
    '4': 'N',
    '5': 'T',
    '6': ':',
    '7': ' ',
    '8': 'n',
    '9': 'o',
    '10': ' ',
    '11': 's',
    '12': 'u',
    '13': 'c',
    '14': 'h',
    '15': ' ',
    '16': 'f',
    '17': 'i',
    '18': 'l',
    '19': 'e',
    '20': ' ',
    '21': 'o',
    '22': 'r',
    '23': ' ',
    '24': 'd',
    '25': 'i',
    '26': 'r',
    '27': 'e',
    '28': 'c',
    '29': 't',
    '30': 'o',
    '31': 'r',
    '32': 'y',
    '33': ',',
    '34': ' ',
    '35': 'o',
    '36': 'p',
    '37': 'e',
    '38': 'n',
    '39': ' ',
    '40': "'",
    '41': '/',
    '42': '/',
    '43': 'p',
    '44': 'u',
    '45': 'b',
    '46': 'l',
    '47': 'i',
    '48': 'c',
    '49': '/',
    '50': 'w',
    '51': 'e',
    '52': 'b',
    '53': 'p',
    '54': 'a',
    '55': 'c',
    '56': 'k',
    '57': '.',
    '58': 's',
    '59': 't',
    '60': 'a',
    '61': 't',
    '62': 's',
    '63': '.',
    '64': 'j',
    '65': 's',
    '66': 'o',
    '67': 'n',
    '68': "'",
    context: {},
    type: 'Error',
    stack: [],
    filename: '',
    message: "ENOENT: no such file or directory, open
'//public/webpack.stats.json'",
    text: 'There was an error',
    level: 'ERROR',
    docsUrl: 'https://gatsby.dev/issue-how-to'
  },
  details: [
    {
      message: '"type" must be one of [GRAPHQL, CONFIG, WEBPACK, PLUGIN]',
      path: [Array],
      type: 'any.only',
      context: [Object]
    }
  ]
}

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @freecodecamp/client@0.0.1 develop: `gatsby develop --inspect=9230 "-H" "0.0.0.0"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @freecodecamp/client@0.0.1 develop script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/gitpod/.npm/_logs/2021-03-29T19_10_58_229Z-debug.log
```

</details>

--- a/client/gatsby-config.js
+++ b/client/gatsby-config.js
@@ -15,6 +15,9 @@ const pathPrefix =
     : '/' + clientLocale;
 
 module.exports = {
+  flags: {
+    DEV_SSR: false
+  },
   siteMetadata: {
     title: 'freeCodeCamp',
     siteUrl: homeLocation

